tests/test_loaddata.py

```diff
@@ -6,18 +6,20 @@
 import unittest
 
 import numpy
+import pytest
 
 from diffpy.utils.parsers import loadData
-from diffpy.utils.tests.testhelpers import datafile
-
-loaddata01 = datafile("loaddata01.txt")
-loaddatawithheaders = datafile("loaddatawithheaders.txt")
 
 
 ##############################################################################
 class TestLoadData(unittest.TestCase):
+    @pytest.fixture(autouse=True)
```

Comment:
Trick to use `@pytest.fixture` within `unittest`

Source: https://stackoverflow.com/a/66222108